main.go

```diff
@@ -25,6 +30,30 @@ var (
 )
 
 func main() {
+	kingpin.MustParse(app.Parse(os.Args[1:]))
+	if bool(*force) == false {
+		files, err := ioutil.ReadDir(".")
+		if err != nil {
+			log.Fatal(err)
+		}
+		reader := bufio.NewReader(os.Stdin)
+		overwrite := false
+		for _, f := range files {
+			if f.Name() == "LICENSE" {
+				fmt.Print("There is already a license present in current directory. Do you want to overwrite it with a new one? [y/N] ")
```

Comment:
It would be nice to have versions added to the tool so you can run `license-up --version` and see what version you are on.

Reply:
I'll make a separate pull request for this ... btw, what version number are we on? 1.0.0?

Reply:
cool ... I'll make a PR right now